pkg/dcgmexporter/pipeline_test.go

```diff
@@ -43,3 +45,90 @@ func TestRun(t *testing.T) {
 	// We'll be testing them more throughly in the e2e tests (e.g: by running prometheus).
```

Comment:
@zwpaper, FYI: The e2e or integration tests were added to the current version in the main branch: https://github.com/NVIDIA/dcgm-exporter/tree/main/tests/integration.